changes/unreleased/Feature-20220711-141818.yaml

```diff
@@ -0,0 +1,4 @@
+kind: Feature
+body: '- Add requireContactMethod and contactMethod optional args to the serviceOwnershipCheck
```

Comment:
can you remove the `- ` from this message.  It will automatically be put in a list in the changlog